src/tox_lsr/test_scripts/runcontainer.sh

```diff
@@ -315,8 +318,11 @@ run_playbooks() {
     setup_vault "$test_dir" "${test_pb_base}.yml"
     # shellcheck disable=SC2086
     pushd "$test_dir" > /dev/null
-    ansible-playbook -vv ${CONTAINER_SKIP_TAGS:-} -i "$inv_file" ${vault_args:-} \
-        -e ansible_playbook_filepath="$(type -p ansible-playbook)" "${test_pb[@]}"
+    for pb in ${test_pb[@]}; do
```

Comment:
I don't think we should change the behavior of doing `tox -e container ... -- tests/tests_*.yml` - if someone really wants to run multiple playbooks with the same invocation of `ansible-playbook` then I think we should assume that they know what they are doing.  A user can use `--parallel 1` to run each playbook separately.

Reply:
Does this commit https://github.com/richm/tox-lsr/pull/1/commits/654d23a6c6eb5109451baf49e4f0e87825ad695d satisfy your suggestion?

Reply:
> Does this commit [654d23a](https://github.com/richm/tox-lsr/commit/654d23a6c6eb5109451baf49e4f0e87825ad695d) satisfy your suggestion?

yes - thanks!